src/lib_c/x86_64-macosx-darwin/c/dyld.cr

```diff
@@ -0,0 +1,4 @@
+lib LibC
```

Comment:
I believe the C header is `mach-o/dyld.h` so it should match it: `c/mach-o/dyld.cr`.